libs/infra-tracing/src/lib/datadog-tracer.ts

```diff
@@ -1,8 +1,26 @@
 import tracer from 'dd-trace'
+import { isPerson } from 'kennitala'
 
 if (process.env.NODE_ENV !== 'development') {
   tracer.init({ logInjection: true }) // initialized in a different file to avoid hoisting.
   tracer.use('express', { blacklist: ['/liveness', '/readiness', '/metrics'] })
+  tracer.use('http', {
+    client: {
+      hooks: {
+        request: (span, req, res) => {
+          const url =
+            res?.url || `http(s)://${req?.getHeader('host')}${req?.path}`
+          if (url) {
```

Comment:
this check is probably not needed anymore since url can't be falsy.

Reply:
Good point, fixed